android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/ProjectConfig.java

```diff
@@ -77,13 +81,7 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeList(platforms);
         dest.writeString(termsOfUse);
 
-        dest.writeBooleanArray(new boolean[]{
-                usesName,
-                webStopped,
-                schedulerStopped,
-                accountCreationDisabled,
-                clientAccountCreationDisabled,
-                accountManager});
+        dest.writeBooleanArray(new boolean[]{usesName, webStopped, schedulerStopped, accountCreationDisabled, clientAccountCreationDisabled, accountManager});
```

Comment:
This is a type of change that makes things worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.